### DISCUSSION TITLE
How to automate flight operations

### DISCUSSION THREAD

**Discussion by jonyMarino** [JSBSim-Team/jsbsim#811] [Q&A]
Hi!
I want to be able to execute flight operations for an airplane; it would be a way of automating things.
For example, having a trigger to start a takeoff or move to a point, or land in any order and number of times.
I guess scripts can be used for that. How? And in that case, is there any way to remove the execution time limit (end=...) from them?

**Comment by seanmcleod** [comment]
Have you looked at some of the scripts that exist in the repo? Some of them demonstrate triggers etc.

> is there any way to remove the execution time limit (end=...) from them?

Doesn't look like it's optional. So for now you'll need to add a very large number.

https://github.com/JSBSim-Team/jsbsim/blob/5ac73c2b555ac7b6b97a1ef43f3f47f1f68a6f0f/src/input_output/FGScript.cpp#L134-L139

Out of interest what are you using JSBSim for? Is this for something AirSim related?
